app/models/activity.rb

```diff
@@ -32,9 +32,21 @@ def revenue_with_cash
     end
   end
 
-  def revenue_without_cash
-    @revenue_without_cash ||= begin
-      OrderRow.where(order: orders.where(paid_with_cash: false)).sum('product_count * price_per_product')
+  def revenue_with_pin
```

Comment:
Don't we just want a 'revenue with pin' column which contains the total revenue with pin? Then you can simply multiply that by 0.019 before entering it into the bookkeeping? If we really want those numbers to have in the system (which I actually don't like due to changing fees) we should make it a configureable number

Reply:
That could work too. Might be a better solution, indeed.